<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsciiOptionsWidget</class>
 <widget class="QWidget" name="AsciiOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3" colspan="2">
    <widget class="KComboBox" name="cbDateTimeFormat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Comment Character:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QCheckBox" name="chbSkipEmptyParts">
     <property name="text">
      <string>Skip empty parts</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QCheckBox" name="chbSimplifyWhitespaces">
     <property name="toolTip">
      <string>Removes the whitespaces from the start and the end, and replaces each sequence of internal whitespaces with a single space.</string>
     </property>
     <property name="text">
      <string>Simplify Whitespaces</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="KComboBox" name="cbSeparatingCharacter">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QFrame" name="frameVectorNamesFromLine">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="sbHeaderLine">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="KComboBox" name="cbCommentCharacter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QCheckBox" name="chbCreateTimestamp">
     <property name="text">
      <string>Create Timestamp </string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QCheckBox" name="chbRemoveQuotes">
     <property name="text">
      <string>Remove Quotes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QComboBox" name="cbDecimalSeparator"/>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QCheckBox" name="chbConvertNaNToZero">
     <property name="text">
      <string>Convert NaN to 0</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="3">
    <widget class="QLineEdit" name="kleVectorNames">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QCheckBox" name="chbHeader">
     <property name="text">
      <string>Names at Line:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="chbCreateIndex">
     <property name="text">
      <string>Create Index</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="lDateTimeFormat">
     <property name="text">
      <string>DateTime Format:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lDecimalSeparator">
     <property name="text">
      <string>Decimal Separator:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="lSeparatingCharacter">
     <property name="text">
      <string>Column Separator:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLabel" name="lVectorNames">
     <property name="text">
      <string>Custom Names:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
